packages/apps/tools/src/components/Global/ModuleExplorer/SidePanel/styles.css.ts

```diff
@@ -0,0 +1,14 @@
+import { style } from '@vanilla-extract/css';
+
+export const containerStyle = style([
+  { display: 'flex', flexDirection: 'column', height: '85vh' },
+]);
+
+export const modulesContainerStyle = style([{ flex: 1, overflow: 'scroll' }]);
+
+export const moduleTitle = style({
+  whiteSpace: 'nowrap',
+  overflow: 'hidden',
+  textOverflow: 'ellipsis',
+  maxWidth: '18rem',
```

Comment:
Is it possible to use some of the already defined values for sizes from the sprinkles?

Reply:
I've refactored it to use values coming from `vars`. The UI/UX still needs to be done for this page, so the values used will most definitely change in the near future.